win_images/win_packaging.ps1

```diff
@@ -33,4 +34,15 @@ Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V-Management-
 # Install WSL, and capture text output which is not normally visible
 $x = wsl --install; Check-Exit 0 1 # wsl returns 1 on reboot required
 Write-Host $x
+
+# Add a WSL configuration file
+# The `kernelBootTimeout` configuration is to prevent CI/CD flakes
+# See https://github.com/microsoft/WSL/issues/13301#issuecomment-3367452109
+$wslConfigPath = "$env:UserProfile\.wslconfig"
+$wslConfigContent = @"
+[wsl2]
+kernelBootTimeout=300000 # 5 minutes
+"@
+Set-Content -Path $wslConfigPath -Value $wslConfigContent
```

Comment:
This is per user right due UserProfile?

I am not sure about the windows build process at all but I am not sure if this here then runs as the same user as our podman tests. Would it be easier to configure in the podman setup instead where we know we run under the correct user https://github.com/containers/podman/blob/main/contrib/cirrus/win-podman-machine-test.ps1?